models/intel/index.md

```diff
@@ -242,5 +242,14 @@ Deep Learning compressed models
 
 
 
+## Question Answering
```

Comment:
There already is a question answering section.

Reply:
Done